fenix/app/src/main/java/org/mozilla/fenix/home/Mode.kt

```diff
@@ -31,34 +20,3 @@ sealed class Mode {
         }
     }
```

Comment:
wondering if, long term, the Mode logic should be in one unified place.

Reply:
Ya, I can imagine refactoring this, but keeping changes to a minimal here.